From baa87c4e45c08b7ded51bdb145c0b21576fc78ad Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 11 Aug 2014 12:26:33 -0400
Subject: [mfd] lpc_ich: Add support for Intel Avoton GPIOs

Message-id: <1407759999-16143-3-git-send-email-prarit@redhat.com>
Patchwork-id: 87597
O-Subject: [RHEL7.1 PATCH BZ 1084134 2/8] mfd: lpc_ich: Add support for Intel Avoton GPIOs
Bugzilla: 1084134
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1084134

commit facd9939403cb5769190054a600474399e776e3a
Author: Vincent Donnefort <vdonnefort@gmail.com>
Date:   Fri Feb 14 15:01:54 2014 +0100

    mfd: lpc_ich: Add support for Intel Avoton GPIOs

    Signed-off-by: Vincent Donnefort <vdonnefort@gmail.com>
    Signed-off-by: Lee Jones <lee.jones@linaro.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/mfd/lpc_ich.c b/drivers/mfd/lpc_ich.c
index ae331cf..165bb0b 100644
--- a/drivers/mfd/lpc_ich.c
+++ b/drivers/mfd/lpc_ich.c
@@ -500,6 +500,7 @@ struct lpc_ich_info lpc_chipset_info[] = {
  [LPC_AVN] = {
   .name = "Avoton SoC",
   .iTCO_version = 1,
+  .gpio_version = AVOTON_GPIO,
  },
  [LPC_COLETO] = {
   .name = "Coleto Creek",
diff --git a/include/linux/mfd/lpc_ich.h b/include/linux/mfd/lpc_ich.h
index 293b062..b2364dd 100644
--- a/include/linux/mfd/lpc_ich.h
+++ b/include/linux/mfd/lpc_ich.h
@@ -39,6 +39,7 @@ enum {
  ICH_V9_GPIO,
  ICH_V10CORP_GPIO,
  ICH_V10CONS_GPIO,
+ AVOTON_GPIO,
 };
 
 struct lpc_ich_info {
-- 
1.7.1